tests/User-test.js

```diff
@@ -0,0 +1,46 @@
+const chai = require('chai');
+const expect = chai.expect;
+const User = require('../src/User');
+
+describe('User', function() {
+  let user;
+
+  beforeEach(function() {
+    user = new User({
+        "id": 1,
+        "name": "Luisa Hane",
+        "address": "15195 Nakia Tunnel, Erdmanport VA 19901-1697",
+        "email": "Diana.Hayes1@hotmail.com",
+        "strideLength": 4.3,
+        "dailyStepGoal": 10000,
+        "friends": [
+          16,
+          4,
+          8
+        ]
+      })
+  });
+
+  it('should be a function', function() {
+    expect(User).to.be.a('function');
+  });
+
+  it('should be an instance of User', function() {
+    expect(user).to.be.an.instanceof(User);
+  });
```

Comment:
I'm not sure we need tests to prove whether it's a function and instance since the rest of the tests imply the same things.

Reply:
Deleted!